google/cloud/bigquery/table.py

```diff
@@ -1669,8 +1684,9 @@ def _validate_bqstorage(self, bqstorage_client, create_bqstorage_client):
         if self._table is None:
             return False
 
-        # The developer is manually paging through results if this is set.
-        if self.next_page_token is not None:
+        # The developer has already started paging through results if
+        # next_page_token is set.
+        if hasattr(self, "next_page_token") and self.next_page_token is not None:
```

Comment:
Just for my education, it looks like attribute `next_page_token` is inherited from "grandparent" class `Iterator` from the core library, which [creates this attribute at init](https://github.com/googleapis/python-api-core/blob/17ff5f1d83a9a6f50a0226fb0e794634bd584f17/google/api_core/page_iterator.py#L183). Is it necessary to check whether this attribute exist or not?

Reply:
This was purely for some failing unit tests where this superclass was mocked out.